broadlink/__init__.py

```diff
@@ -1063,6 +1063,18 @@ def set_state(self, state):
         cmd = '{"pwr":%d}' % (1 if state == "ON" or state == 1 else 0)
         self.send_command(cmd)
 
+    def set_brightness(self, state):
+        cmd = '{"brightness":%d}' % state
+        self.send_command(cmd)
+
+    def set_power(self, state):
+        self.set_brightness(100)
+        self.set_state(state)
+
+    def check_power(self):
+        return self.get_state()
```

Comment:
The same for this one. Why a method just to call another method that already exists?

Reply:
I wrote this as a hook to broadlink_cli . Am I wrong?

Reply:
Same here.

Reply:
Same here.